usaspending_api/accounts/models.py

```diff
@@ -6,40 +6,70 @@
 # Table #3 - Treasury Appropriation Accounts.
 class TreasuryAppropriationAccount(DataSourceTrackedModel):
     treasury_account_identifier = models.AutoField(primary_key=True)
-    tas_rendering_label = models.CharField(max_length=22, blank=True, null=True)
+    tas_rendering_label = models.CharField(max_length=50, blank=True, null=True)
     allocation_transfer_agency_id = models.CharField(max_length=3, blank=True, null=True)
     agency_id = models.CharField(max_length=3)
     beginning_period_of_availability = models.CharField(max_length=4, blank=True, null=True)
     ending_period_of_availability = models.CharField(max_length=4, blank=True, null=True)
     availability_type_code = models.CharField(max_length=1, blank=True, null=True)
     main_account_code = models.CharField(max_length=4)
     sub_account_code = models.CharField(max_length=3)
-    gwa_tas = models.CharField(max_length=21, blank=True, null=True)
-    gwa_tas_name = models.CharField(max_length=300, blank=True, null=True)
-    agency_aid = models.CharField(max_length=3, blank=True, null=True)
-    agency_name = models.CharField(max_length=100, blank=True, null=True)
-    admin_org = models.CharField(max_length=6, blank=True, null=True)
-    admin_org_name = models.CharField(max_length=100, blank=True, null=True)
-    fr_entity_type_code = models.CharField(max_length=4, blank=True, null=True)
+    # gwa_tas = models.CharField(max_length=21, blank=True, null=True)
```

Comment:
I agree, but bernard and alex left them in there and I wasn't sure if there was a need or not.